src/vs/workbench/contrib/chat/browser/chatDragAndDrop.ts

```diff
@@ -239,11 +244,9 @@ export class ChatDragAndDrop extends Themable {
 			return this.resolveSymbolsAttachContext(data);
 		}
 
-		// Removing HTML support for now
-		// if (containsDragType(e, 'text/html')) {
-		// 	const data = e.dataTransfer?.getData('text/html');
-		// 	return data ? this.resolveHTMLAttachContext(data) : [];
-		// }
+		if (!containsDragType(e, DataTransfers.INTERNAL_URI_LIST) && containsDragType(e, 'text/uri-list') && ((containsDragType(e, 'text/html') || containsDragType(e, 'text/plain')))) {
```

Comment:
We have constants for these mime types in `Mimes`. This makes it easier to track down where they are used